components/docs-chef-io/content/automate/ha_disaster_recovery_setup_AWS.md

```diff
@@ -0,0 +1,157 @@
++++
+title = "Disaster Recovery Setup"
+
+draft = false
+
+gh_repo = "automate"
+[menu]
+  [menu.automate]
+    title = "Disaster Recovery for AWS Deployment"
+    parent = "automate/deploy_high_availability/disaster_recovery"
+    identifier = "automate/deploy_high_availability/ha_disaster_recovery_setup_AWS.md Disaster Recovery"
+    weight = 200
++++
+
+{{< warning >}}
+{{% automate/ha-warn %}}
+{{< /warning >}}
+
+## Setup Disaster Recovery Cluster For AWS Deployment
+
+Recovery Point Objective (RPO) is the maximum acceptable amount of time since the last data recovery point, if an RPO of 1 to 24 hours is acceptable then using a typical backup and restore strategy for your disaster recovery plan is recommended.
+Typically these two clusters should be located in different data centers or cloud provider regions.
+
+### Requirements
+
+1. Two identical clusters located in different cloud provider regions.
```

Comment:
Should it be "different regions of the cloud provider"?

"Two identical clusters located in different AWS regions"